pandas/tests/indexing/test_at.py

```diff
@@ -97,6 +99,24 @@ def test_at_setitem_categorical_missing(self):
 
         tm.assert_frame_equal(df, expected)
 
+    @pytest.mark.parametrize("row",
+                             (Timestamp("2019-01-01"),
+                              to_datetime("2019-01-01"),
```

Comment:
this is the same as the Timestamp argument

Reply:
Yes, this can be removed in favor of the `Timestamp` param above

Reply:
Apologies I misunderstood earlier. I thought you were suggesting using `Timestamp()` as another test to be added

Reply:
Yeah, sorry. This wasn't clear on my side